openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1343,6 +1343,9 @@ This specification defines the following Credential Issuer Metadata:
 * `display`: OPTIONAL. Array of objects, where each object contains display properties of a Credential Issuer for a certain language. Below is a non-exhaustive list of valid parameters that MAY be included:
   * `name`: OPTIONAL. String value of a display name for the Credential Issuer.
   * `locale`: OPTIONAL. String value that identifies the language of this object represented as a language tag taken from values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
+  * `logo`: OPTIONAL. Object with information about the logo of the Credential Issuer with the following non-exhaustive list of parameters that MAY be included:
+    * `uri`: REQUIRED. String value that contains a URI where the Wallet can obtain a logo of the Credential Issuer. The Wallet needs to determine the scheme, since the URI value could use `https:` scheme, `data:` scheme, etc.
+    * `alt_text`: OPTIONAL. String value of an alternative text of a logo image.
```

Comment:
```suggestion
  * `logo`: OPTIONAL. Object with information about the logo of the Credential Issuer. Below is a non-exhaustive list of parameters that MAY be included:
    * `uri`: REQUIRED. String value that contains a URI where the Wallet can obtain the logo of the Credential Issuer. The Wallet needs to determine the scheme, since the URI value could use the `https:` scheme, the `data:` scheme, etc.
    * `alt_text`: OPTIONAL. String value of the alternative text for the logo image.
```